src/ratd.md

```diff
@@ -0,0 +1,3 @@
+# RATS Conceptual Message Collection Daemon
```

Comment:
Minor Question: Should the name of the file be ratsd.md ?

Reply:
Yes, because this file is about `ratsd`? I could call it "rats-conceptual-message-collection-daemon.md" but that's kinda  long, and the file names are not visible to the reader so having it named after the the repo/executable it's describing seems preferable.

Reply:
oh, I see, yes, that's a typo. Will fix.